### ISSUE TITLE
focusWindowWest() very slow on OS X 10.9.5

### ISSUE THREAD

**Issue by posto** [Hammerspoon/hammerspoon#277]
The ```focusWindowWest()``` is very slow on my machine (focus east, north and west are also slow, for that matter).
It always takes about 10seconds to move the focus, which makes it ususable.

Can I somehow help to debug this issue?

Btw, ```hs.hints.windowHints``` is also very slow, might be somehow related.

**Comment by cmsj** [comment]
@posto Is it also this slow after a reboot? I've seen instances where the OS X Accessibility API calls take a while to return, but only after the machine has been up for quite a while.

**Comment by posto** [comment]
Just restarted. Still happens, takes 6 sec to switch :(

**Comment by dktn** [comment]
Same here: focus and hints take 6 seconds.

**Comment by cmsj** [comment]
In these situations, it's worth checking Activity Monitor for any processes that are listed as Not Responding.

The reason being, when we use the Accessibility API to go off and fetch window information, the replies actually come from the apps themselves, so if one of them has hung, or is just really busy, there will be a delay in processing the functions.

**Comment by dktn** [comment]
It is not a case, none of the application is busy nor not responding. Processor idle 97-98%.
In contrast hs.application.launchOrFocus and all hs.grid operations work instantly.

**Comment by cmsj** [comment]
I think this is a duplicate of #233, so I'm closing this issue.

**Comment by lowne** [comment]
("crossposting" for completeness)
After having updated to 0.9.40 or later,  as soon as you (if you still) notice sluggishness or excessive delay (upon calling `windowHints`, `layout`, etc), please open the Console and run this:
```lua
hs.window._timed_allWindows()
```
(If you get no output, try again a bunch of times (up arrow+return) until you (hopefully) do)

Then please paste the output in issue #233.
